pom.xml

```diff
@@ -1,76 +1,191 @@
-
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>com.diffblue</groupId>
   <artifactId>deeptestutils</artifactId>
-  <version>1.0.0</version>
+  <version>0.0.1</version>
   <packaging>jar</packaging>
 
   <name>Deeptest-utils</name>
-
+  <description>Utilities used in tests automatically generated by Diffblue Deeptest.</description>
+  <url>diffblue.com</url> 
+  <!-- must declare the license(s) used for distributing your components. -->
+  <!-- @todo review license - apache vs MIT vs... -->
```

Comment:
Where is the license provided? There should be a license file in the root of the repo.

Reply:
Done in #5